### DISCUSSION TITLE
Google Calendar Name

### DISCUSSION THREAD

**Discussion by bklein1979** [add2cal/add-to-calendar-button#403] [Q&A]
I have been looking for a solution to this, but I am not sure it is documented. Is there a way to pass a name for Google Calendar to use when subscribing to a calendar? Right now it is the webcal link, I have other calendars that I have subscribed to that have a name that I did not set, but rather was set by the subscribe button.

**Comment by jekuer** [comment]
Afaik, this is not possible. It would only work if the calendar address, you provide, is linked to a Google calendar - in this case, the Google calendar of your user can fetch the name.
In case somebody know a generic solution, feel free to mention it here!
